### PR TITLE
Create salt states to manage rails server and resque worker like services

### LINKED ISSUE #9: VMs aren't running services
We should manage simple nohup scripts on the VMs to start core and worker services with salt.

### PR DESCRIPTION
**Pull request by cdarwin**
Fixes #9

### CODE REVIEW

File: salt/roots/worker/service.sls [hunk 1]
```diff
@@ -0,0 +1,11 @@
+resque worker service:
+  cmd.run:
+    - name: 'nohup bundle exec rake resque:work > /vagrant/log/resque.log 2> /vagrant/log/resque.err < /dev/null &'
+    - cwd: /vagrant
+    - unless: pgrep -fa resque
+    - env:
+      - DB_HOST: 192.168.69.20
```
Comment (akira) [inline salt/roots/worker/service.sls]: Should we also add here?
```DOCKER_URL=unix:///run/docker.sock```

File: README.md [hunk 2]
```diff
@@ -21,18 +21,12 @@ luser@lolcathost:perfocmanceci-core $ git subtree add --prefix salt/formulas/doc
 
 ### Vagrant
 
-We should strive to provide a Vagrantfile meeting the minimum requirements to
+We have attempred to provide a Vagrantfile meeting the minimum requirements to
```
Comment (akira) [inline README.md]: sp :smile:

**Comment by akira** [comment]
Awesome!  I think this will work well

File: salt/roots/worker/service.sls (hunk 1 above)
Comment (cdarwin) [inline salt/roots/worker/service.sls]: It hasn't needed it in my experience. I don't see anywhere that uses that environment variable, either. Oh, it's part of the swipely/docker-api#86 library that we use. I'd rather address this in an additional feature (#20) of pulling more of this out of the states and into pillar data.

Comment (akira) [inline salt/roots/worker/service.sls]: Maybe the new upgraded version? (#19) Anyways, sounds good, I'll merge.

Comment (cdarwin) [inline salt/roots/worker/service.sls]: Ah, I'll bet you're right! I've pulled your branch for testing now. I rebased the changeset on the upstream master, so I'll push any additional fixups to my own remote. I will let you resolve your upstream branch accordingly.

**Pull request merged**